kolibri/plugins/learn/assets/src/views/TopicsContentPage.vue

```diff
@@ -385,37 +389,42 @@
         if (!this.content) {
           return Promise.resolve();
         }
-        // Fetch the next content
-        const nextPromise = ContentNodeResource.fetchNextContent(this.content.parent, {
-          topicOnly: true,
-        }).then(nextContent => {
-          // This may return the immediate parent if nothing else is found so let's be sure
-          // not to assign that
-          if (nextContent && this.content.parent !== nextContent.id) {
-            this.nextContent = nextContent;
-          }
-        });
+        const fetchGrandparent = this.content.ancestors.length > 1;
```

Comment:
It would be good to keep the naming consistent with camel case

Reply:
I guess I was thinking of 'grandparent' as one word :)